docs/agents/ifm_sbn246_flowmeter.rst

```diff
@@ -0,0 +1,94 @@
+.. highlight:: rst
+
+.. _ifm_sbn246_flowmeter:
+
+======================
+SBN246 Flowmeter Agent
+======================
+
+The SBN246 Flowmeter Agent is an OCS Agent which monitors flow in liters per minute
+and temperature in Celsius of the cooling loop of the DRs installed at the site.
+Monitoring is performed by connecting the flowmeter device to a DAQ IO-Link readout device from
+the same company, the querying of flowmeter data is done via HTTP requests to that DAQ device.
+
+.. argparse::
+    :filename: ../socs/agents/ifm_sbn246_flowmeter/agent.py
+    :func: add_agent_args
+    :prog: python3 agent.py
+
+DAQ IO-Link Device Network Setup
+--------------------------------
+After plugging into the IoT port on your IFM Electronic AL1340 DAQ IO-Link device,
+the IP address of the IO-Link device must be configured using the LR Device software.
+On the LR Device software, click the 'read from device' on the upper right (leftmost
+IOLINK button). Once the software has found your IO-Link device, click on the device name
+(AL1340) on the column on the left side (not the ports that may say P1, P2... or the name
+of the device connected to any of the 4 ports on the IO-Link device). You should now be
+able to change the IP address to by clicking the Network (DHCP) button.
+
+ - `LR Device Software <https://www.ifm.com/de/en/download/LR_Device>`_
```

Comment:
I think we do need to go in and set it to DHCP? Memory's already a little foggy now but I seem to recall the LR software finds the device, and assigns the factory setting ip address until we go into network parameter and toggle it to DHCP even for the IoT. And I think that's how you got it from 169.254.X.X to 10.10.10.159? Am I remembering that correctly? I also think that was our big argument for PR'ing the http requests version of this agent since we have to do a network configuration via the IoT port for this device anyway

Reply:
I didn't change anything for the IoT ports network settings, so unless you did I think it's DHCP by default. It's sort of hard to tell from the documentation, which says:

> To configure the IP settings with DHCP, a DHCP server has to be active in the IT network. If
> no DHCP server can be reached in the IT network, an IP address is automatically assigned to
> the IoT port with the Zeroconfig protocol (address range: → Factory settings (→ p. 91)).

If it is DHCP by default, then yeah, we don't need to run the LR Software and it should just get an address and work. Can we test by factory resetting the device we have in the lab?